org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/smap/DebugSourceInstallingCompilationParticipant.java

```diff
@@ -185,8 +185,26 @@ public void buildStarting(BuildContext[] files, boolean isBatch) {
 	}
 
 	protected List<IFile> findGeneratedJavaClassFiles(IJavaElement element) {
-		Region region = new Region();
-		region.add(element);
+		IRegion region = new IRegion() {
```

Comment:
why not javacore.newRegion?

Reply:
just was not aware of that. will refactor it.

Reply:
updated